README.md

```diff
@@ -1,405 +1,427 @@
+- [jpo-ode](https://github.com/usdot-jpo-ode/jpo-ode)
+- [jpo-conflictvisualizer](https://github.com/usdot-jpo-ode/jpo-conflictvisualizer)
+
+**Ongoing Efforts**
+This feature is under active development. This is a joint effort involving combining the features of the existing CIMMS conflictvisualizer tools with the CVManager components, to enable connected vehicle and intersection analysis in one application.
+
+One of the major features which is under active development is the combined CVManager - ConflictVisualizer API. This can currently be found under the services/intersection-api folder.
+
+#### Github Token
+
+A GitHub token is required to pull artifacts from GitHub repositories. This is required to obtain the jpo-ode jars and must be done before attempting to build this repository.
+
+1. Log into GitHub.
+2. Navigate to Settings -> Developer settings -> Personal access tokens.
+3. Click "New personal access token (classic)".
+   1. As of now, GitHub does not support Fine-grained tokens for obtaining packages.
+4. Provide the name "jpo_conflictmonitor"
+5. Set an expiration date
+6. Select the read:packages scope.
+7. Click "Generate token" and copy the token.
+8. Set this token as the MAVEN_GITHUB_TOKEN environment variable in the .env file (root and ./services/intersection-api/.env)
+
+### Feature Flags
+
+This application has the ability to disable certain features based on environment variables. For each of these variables, the feature will be enabled if the variable is anything but 'false'. These features include:
+
+- ENABLE_RSU_FEATURES: if 'false', disable all RSU-specific features, including map, RSU data, RSU configuration, and RSU organization linking.
+- ENABLE_INTERSECTION_FEATURES: if 'false', disable all intersection-specific features, including intersection map, intersection dashboard, and intersection admin pages.
+- ENABLE_WZDX_FEATURES: if 'false', disable all wzdx-specific features, including WZDx data on the main map.
+
+These variables will disable UI pages/visuals, UI features, and API endpoints.
+
+#### Local Development
+
+Ease of local development has been a major consideration in the integration of intersection data into the CVManager application. Through the use of public docker images and sample datasets, this process is relatively simple. The services required to show intersection data on the CVManager webapp are:
+
+- kafka
+  - Base kafka image used to supply required topics to the intersection api
+- kafka_init
+  - Kafka topic creation image, to create required topics for the intersection api
+- MongoDB
+  - Base MongoDB image, with sample data, used to supply data to the intersection api
+
+**Intersection API Submodules**
+The Intersection API uses submodules to reference the ConflictMonitor, ODE, and other services. These submodules need to be initialized and updated before the API can be built and run locally. Run the following command to initialize the submodules:

```

Comment:
Question (non-blocking): Is this statement accurate? I'm only seeing the asn1_codec repository as a git submodule.

Reply:
asn1_codec is now the only submodule. Updated readme